en/flying/led_meanings.md

```diff
@@ -41,3 +48,27 @@ this flight.
 Attach your autopilot to a Ground Control Station to verify what the problem is.
 If you have completed the setup process and autopilot still appears as red and flashing, there may be another error.
 
+## Status LED (FMUv5 and later) {#status_led}
+
+Three *Status LEDs* were added to the *Pixhawk Series* in Pixhawk 4 ([FMUv5](../flight_controller/pixhawk_series.md#fmu-versions)) to provide status for the PX4IO and FMU SoC. 
+They indicate power, bootloader mode and activity, and errors.
+
+The labeling and purpose of the LEDs on the PX4 reference board (HolyBro Pixhawk 4) for bootloading and when in normal use is given below:
```

Comment:
Are the labels likely to be the same for any board? I assume not, so have specified HolyBro.